Formula/wxpython.rb

```diff
@@ -16,51 +15,45 @@ class Wxpython < Formula
     sha256 cellar: :any, catalina:       "363cff6b220e1d12e1075e3dfbb580eab83c0fea65f3c4f398d2d6eba80c5145"
   end
 
-  depends_on "freetype"
-  depends_on "jpeg-turbo"
-  depends_on "libpng"
-  depends_on "libtiff"
```

Comment:
Did we remove these because it's actually using bundled versions of these?

Reply:
I think most dependencies were originally used to build `pillow` and `wxwidgets`.